types/types.go

```diff
@@ -49,6 +49,14 @@ func ToPbECGroupElement(el *ECGroupElement) *pb.ECGroupElement {
 	return &x
 }
 
+func CmpECGroupElements(a, b *ECGroupElement) bool {
+	if a.X.Cmp(b.X) == 0 && a.Y.Cmp(b.Y) == 0 {
```

Comment:
I see the first comment coming :)

Reply:
Well, you figured this one yourself :)

Why not simply `return a.X.Cmp(b.X) == 0 && a.Y.Cmp(b.Y) == 0` ?